packages/app-mobile/components/screens/ConfigScreen/ConfigScreen.tsx

```diff
@@ -830,16 +830,19 @@ class ConfigScreenComponent extends BaseScreenComponent<ConfigScreenProps, Confi
 		}
 
 		let screenHeadingText = _('Configuration');
+		let showSearchButton = true;
+
 		if (currentSectionName) {
 			screenHeadingText = Setting.sectionNameToLabel(currentSectionName);
+			showSearchButton = screenHeadingText !== _('Plugins');
 		}
```

Comment:
```suggestion
		if (currentSectionName) {
			screenHeadingText = Setting.sectionNameToLabel(currentSectionName);
			showSearchButton = currentSectionName !== 'plugins';
		}
```

Consider comparing with the section ID instead — section IDs shouldn't change, while the `'Plugins'` user-facing string might be updated in the future.

Reply:
okay